server/meshmodel/kubernetes/relationships/hierarchical_inventory.json

```diff
@@ -210,7 +210,7 @@
                   "template",
                   "spec",
                   "containers",
-                  "_",
+"resources",                   "_",
```

Comment:
Will you re-check this, I don't think this is accurate?

Reply:
Yes, this is by mistake. I will update the PR soon.

Reply:
Corrected.